packages/shared/components/modals/NodeActionsMenu.svelte

```diff
@@ -56,7 +56,7 @@
         } else {
             await togglePrimaryNodeInClientOptions(node)
             if (Platform.isFeatureFlagEnabled('governance')) {
-                await registerProposalsFromPrimaryNode()
+                await registerProposalsFromNode(node)
```

Comment:
If we are now registering all proposals at login and also when we add a new node, we do not need to register a proposal at changing a primary node.